about.html

```diff
@@ -0,0 +1,153 @@
+    <title>Interior Design Symposium.</title>
+</head>
+<body>
+    <section class="main-section">
+    <header class="navbar">
+        <nav class="navbar-1">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <a class="nav-items-11" href="#">English</a>
+            <a class="nav-items-12" href="#">My Page</a>
+            <a class="nav-items-13" href="#">Logout</a>
+        </nav>
+        <nav class="navbar-2">
+            <h2 class="nav-logo">SUAVE INTERIORS</h2>
+            <a class="nav-items" href="./index.html">Home</a>
+            <a class="nav-items" href="./about.html">About</a>
+            <a class="nav-items" href="#">Program</a>
+            <a class="nav-items" href="#">News</a>
+            <a class="nav-items-2" href="#">dc Campaign</a>
+        </nav>
+        <nav class="nav-list">
+            <a  class="nav-items" href="./index.html">Home</a>
+            <a  class="nav-items" href="./about.html">About</a>
+            <a class="nav-items" href="#">Program</a>
+            <a class="nav-items" href="#">News</a>
+            <a class="nav-items-2" href="#">dc Campaign</a>
+        </nav>
+            <nav class="bars">
+              <div class="bars-one"></div>
+              <div class="bars-one"></div>
+              <div class="bars-one"></div>
+            </nav>
+            </nav>
+        </header>
+        <section id="about">
+         <div class="about-title1">
+          <p class="about-firstline">"Hello! Interior Design Lovers"</p>
+            <h3 class="about-header">We bring to you the 2022<br> 
+              Interior design Symposium live in Uganda.
+            </h3>
+         </div>
+         <div class="about-quote">
+            <p class="about-quote-1">In this years symposium which will be both online and free for everyone to attend.
+                We have a vast number of panel speakers, lots of interesting prices and scholarships to give to the lucky winners.
+            </p>
+         </div>
+         <div class="about-details">
+            <p class="about-details-1">Reach us via our socials<br>
+                for more about the symposium!</p>
+            <p class="about-details-2">facebook @Interiordesign <br>
+                                       twitter @interiordesign.
+
+            </p>
+         </div>
+       </section>
+    </section>
+        
+    </div>
+    <section id="about">
+    <div class="about-container1">
+    <h4 class="about-mp">Interior Design<br>
+         2022 Logo</h4>
+    <hr class="hr-title">
+        <ul class="">
+            <div class="about-box">
+            <p class="about-box-2">The logo of this symposium was designed by Phillip Hall in 2016.</p>
+            <div class="logo-img">
+            <h2>SUAVE INTERIORS.</h2>
+            
+        </div>
+        
+        </ul>
+    </div>
+    </section>
```

Comment:
- [ ] Kindly make sure that You follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), For Mobile screen size the text are overlapping please try to adjust Layout: composition and space between elements.

Your Project Look likes this : 
![C1](https://user-images.githubusercontent.com/93598189/191457008-3f61f4b1-1fbb-4414-8762-8c65c17d28e0.PNG)

The Design Template Look likes this : 
![Ca1](https://user-images.githubusercontent.com/93598189/191457239-36b2fe90-b662-45b7-9684-1a4ef18b9e87.PNG)